README.md

```diff
@@ -130,10 +130,10 @@ This dbt package is dependent on the following dbt packages. Please be aware tha
 ```yml
 packages:
     - package: fivetran/shopify
-      version: [">=0.10.0", "<0.11.0"]
+      version: [">=0.11.0", "<0.14.0"]
 
     - package: fivetran/shopify_source
-      version: [">=0.10.0", "<0.11.0"]
+      version: [">=0.11.0", "<0.13.0"]
```

Comment:
Should this be the range that dbt_shopify uses?https://github.com/fivetran/dbt_shopify/blob/main/packages.yml